docs/framework/network-programming/how-to-download-files-with-ftp.md

```diff
@@ -1,55 +1,77 @@
 ---
-title: "How to: Download Files with FTP"
-ms.date: "03/30/2017"
+title: "How to: Download files with FTP"
+description: "This article shows a sample of how to download a file from an FTP server."
+ms.date: "06/25/2018"
+dev_langs: 
+  - "csharp"
+  - "vb"
 ms.assetid: 892548b8-954a-4f6a-9bca-2ae620c3700f
-author: "mcleblanc"
-ms.author: "markl"
-manager: "markl"
 ---
-# How to: Download Files with FTP
-This sample shows how to download a file from an FTP server.  
-  
-## Example  
-  
-```csharp  
-using System;  
-using System.IO;  
-using System.Net;  
-using System.Text;  
-  
-namespace Examples.System.Net  
-{  
-    public class WebRequestGetExample  
-    {  
-        public static void Main ()  
-        {  
-            // Get the object used to communicate with the server.  
-            FtpWebRequest request = (FtpWebRequest)WebRequest.Create("ftp://www.contoso.com/test.htm");  
-            request.Method = WebRequestMethods.Ftp.DownloadFile;  
-  
-            // This example assumes the FTP site uses anonymous logon.  
-            request.Credentials = new NetworkCredential ("anonymous","janeDoe@contoso.com");  
-  
-            FtpWebResponse response = (FtpWebResponse)request.GetResponse();  
-  
-            Stream responseStream = response.GetResponseStream();  
-            StreamReader reader = new StreamReader(responseStream);  
-            Console.WriteLine(reader.ReadToEnd());  
-  
-            Console.WriteLine("Download Complete, status {0}", response.StatusDescription);  
-  
-            reader.Close();  
-            response.Close();    
-        }  
-    }  
-}  
-```  
-  
-## Compiling the Code  
- This example requires:  
-  
--   References to the **System.Net** namespace.  
-  
-## Robust Programming  
-  
-## .NET Framework Security
+# How to: Download files with FTP
+
+This sample shows how to download a file from an FTP server.
+
+## Example
+
+```csharp
+using System;
+using System.IO;
+using System.Net;
+
+namespace Examples.System.Net
+{
+    public class WebRequestGetExample
+    {
+        public static void Main ()
+        {
+            // Get the object used to communicate with the server.
+            FtpWebRequest request = (FtpWebRequest)WebRequest.Create("ftp://www.contoso.com/test.htm");
+            request.Method = WebRequestMethods.Ftp.DownloadFile;
+
+            // This example assumes the FTP site uses anonymous logon.
+            request.Credentials = new NetworkCredential("anonymous","janeDoe@contoso.com");
+
+            FtpWebResponse response = (FtpWebResponse)request.GetResponse();
+
+            Stream responseStream = response.GetResponseStream();
+            StreamReader reader = new StreamReader(responseStream);
+            Console.WriteLine(reader.ReadToEnd());
+
+            Console.WriteLine("Download Complete, status {0}", response.StatusDescription);
```

Comment:
These Console.WriteLine statements with composite formatting strings could be done as interpolated strings in C# and VB if you'd like to make the changes. Here, `Console.WriteLine($"Download Complete, status {response.StatusDescription}");

Reply:
are we adding notes about C# version needed when adding interpolated strings?

Reply:
Sorry, @mairaw. I missed your question. No, we haven't been, other than in the C# and VB interpolated string docs themselves.